texts/depositing/data-overview.md

```diff
@@ -1,4 +1,6 @@
 ---
 title: Data overview
 redirect: Review data
+tooltip-non-compliant: The number of documents that were deposited outside the 92 day window. Data is calculated from the deposit date that CORE collects from the metadata record and from the publication date, which is a combination from other repository data and Crossref.
```

Comment:
I don't like the speech from the third person but it is endless. I hope we will sort it out soon and will update the messages properly.